egs2/chime2/enh1/run.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+# NOTE1: The reference signals in spk1.scp in train and devel subsets are reverberated.
+# NOTE2: No reference signals are available for test subsets.
+
+train_set=train
+valid_set=devel
+test_sets="test"
+
+./enh.sh \
+    --train_set "${train_set}" \
+    --valid_set "${valid_set}" \
+    --test_sets "${test_sets}" \
+    --fs 16k \
```

Comment:
This is the same as default value. Maybe not necessary to list here.

Reply:
done